README.md

```diff
@@ -1 +1,79 @@
+simulations) but necessary for uploading your own data.
+
+### Interactive cell factory design and omics data integration
+
+In the [Interactive Map](https://app.dd-decaf.eu/app/pathwayvis) we utilize metabolic pathway maps as an intuitive user interface for model simulation and data analysis, giving biologists the ability to modify models in a visual way instead of writing programming code. For example, reactions can be deleted from the host organism.
+
+<img src="gifs/knockout.gif" alt="Reaction knockout" width="650">
+
+All modifications can be undone (for example the reaction deletion).
+
+<img src="gifs/undo-knockout.gif" alt="Reaction knockout" width="650">
+
+The side panel on the left provides the user with additional information (for example the medium composition) and the ability to select different model simulation methods. The default simulation method, [pFBA](http://cobramethods.wikidot.com/pfba) (_parsimonious flux balance analysis_), will compute a flux distribution that maximizes growth yield and minimizes the cost of enzyme expression simultaneously. Descriptions of the other available methods will follow shortly.
+
+<img src="gifs/side-panel-medium-simulation-methods.gif" alt="Reaction knockout" width="650">
+
+Computationally more extensive methods can be chosen to get uncertainty estimates for the predicted fluxes. For example, pFBA-FVA combines [pFBA](http://cobramethods.wikidot.com/pfba) with [FVA](http://cobramethods.wikidot.com/flux-variability-analysis) (_flux variability analysis_). Uncertain flux are highlighted using transparency.
+
+<img src="gifs/pfba-fva.gif" alt="Reaction knockout" width="650">
+
+The side panel furthermore provides a selection of different pathway maps the user can explore other parts of metabolism for example (amino acid biosynthesis).
+
+<img src="gifs/amino-acids-pathway.gif" alt="Select amino acid biosynthesis map" width="650">
+
+You can search the map by pressing Ctrl-F (Command-F on Mac). _The arrow buttons for cycling through the matches are currently not visible. This will be fixed soon._
+
+<img src="gifs/searching-on-pathway.gif" alt="Select amino acid biosynthesis map" width="650">
+
+Furthermore native and heterologous reactions can be added on the fly which in combination with reaction deletions (see above) facilitates manual strain design.
+
+<img src="gifs/add-reaction.gif" alt="Add pyruvate carboxylase" width="650">
+
+The application furthermore allows the user to analyze and integrate data with models (see below for how to upload data to the platform). 
+
+__TODO: add gif for data-driven card and animation__
+
+### Computational cell factory design
+
+In the [Pathways](https://app.dd-decaf.eu/app/pathways) application you can predict metabolic routes to products that your can host can not natively produce. You can share pathways with the [Interactive Map](https://app.dd-decaf.eu/app/pathwayvis) application for further inspection. 
```

Comment:
"your can host can" - no need for first `can` IMO.

Reply:
done